stable/dynamic-gateway-service/static/init-admin-user.sh

```diff
@@ -0,0 +1,23 @@
+if [[ -f /init/admin-user-secret/password-hashed ]] ; then
```

Comment:
Per Slack conversation, we should check for file emptiness here, as an empty secret would result in an inaccessible admin user.